pkg/debugger/server.go

```diff
@@ -42,14 +42,14 @@ func (ds DebugConfig) GetHandlers() map[string]http.Handler {
 // NewDebugConfig returns an implementation of DebugConfig interface.
 func NewDebugConfig(certDebugger *certificate.Manager, xdsDebugger XDSDebugger, meshCatalog catalog.MeshCataloger,
 	proxyRegistry *registry.ProxyRegistry, kubeConfig *rest.Config, kubeClient kubernetes.Interface,
-	kubeController k8s.Controller, msgBroker *messaging.Broker) DebugConfig {
+	computeClient compute.Interface, msgBroker *messaging.Broker) DebugConfig {
 	return DebugConfig{
-		certDebugger:   certDebugger,
-		xdsDebugger:    xdsDebugger,
-		meshCatalog:    meshCatalog,
-		proxyRegistry:  proxyRegistry,
-		kubeClient:     kubeClient,
-		kubeController: kubeController,
+		certDebugger:  certDebugger,
+		xdsDebugger:   xdsDebugger,
+		meshCatalog:   meshCatalog,
```

Comment:
can we remove the meshcatalog here as well?

Reply:
Discussed offline, will create a follow up task for this.